make.js

```diff
@@ -144,8 +144,13 @@ target.test = function() {
 
     var suiteArg = process.argv[4];
     var suiteType = suiteArg || 'L0';  
-    var testsSpec = path.join(buildPath, "/**/Tests", suiteType + ".js");
-    run("mocha " + testsSpec, true);
+
+    var taskArg = process.argv[5];
+    var taskType = taskArg || '**';
+
+    // the tests may be under _build or _common
+    var testsSpec = path.join(__dirname, '_[bc]*', taskType, 'Tests', suiteType + ".js");
+    run(`mocha ${testsSpec}`, true);
```

Comment:
the `_[bc]*` pattern is not ideal. i played with it a little and couldn't figure out anything better given that's its in a separate directory. if we pass two separate patterns to mocha, then it complains if one is unmatched.